### PR TITLE
fix(ci): feature-deploy url's missing

### PR DESCRIPTION
**Pull request by peturgq**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -623,7 +623,7 @@ jobs:
         run: |
           ./scripts/build-docker-container.sh $DOCKER_TAG
           echo "COMMENT<<EOF" >> $GITHUB_ENV
-          printf "Affected services are: ${{needs.prepare.outputs.IMAGES}}\n" >> $GITHUB_ENV
+          printf "Affected services are: `${{needs.prepare.outputs.IMAGES}}`\n" >> $GITHUB_ENV
```
Comment (busla) [inline .github/workflows/push.yml:626]: I dont think backticks are applicable here, they are (were, deprecated now) used for command substition, e.g. `foo=$(echo "bar")`

The `${{needs.prepare.outputs.IMAGES}}` variable is a string.

Comment (krrrrrrrrrrr) [inline .github/workflows/push.yml:626]: Why not `echo`? Prevents breakage in case `IMAGES` ever includes `%`

**Review by busla — CHANGES_REQUESTED** [review]
backticks in bash consideration

File: .github/workflows/push.yml (hunk 1 above)
Comment (krrrrrrrrrrr) [reply .github/workflows/push.yml:626]: I like to use `'` for delimination in these cases

Comment (busla) [reply .github/workflows/push.yml:626]: that would end up as a literal string and would output the variable name instead of the value

Comment (busla) [reply .github/workflows/push.yml:626]: probably because of the `\n` ?

Comment (busla) [reply .github/workflows/push.yml]: backticks are still there

**Review by busla — APPROVED** [review]
LGTM

**Review by busla — APPROVED** [review]
LGTM

**Pull request merged**